nuxt.config.ts

```diff
@@ -101,6 +101,9 @@ export default defineNuxtConfig({
         || 'https://proxy.calendar.library.ucla.edu/',
       esTempIndexPrefixLibguides: process.env.ES_TEMP_INDEX_PREFIX_LIBGUIDES || '',
       esTempIndexLibguides: '',
+      gtm: {
+        id: 'GTM-T2SXV2'
```

Comment:
Can we make this an .ENV variable? Or is this a public ID we don't need to hide?

Reply:
The tag was not hidden in Nuxt2 PR. And from my understanding of GTM, the ID has to be exposed on pages to be tracked.

Reply:
Okay great, thanks for checking!